.meta/sinks/papertrail.toml

```diff
@@ -0,0 +1,34 @@
+[sinks.papertrail]
+title = "Papertrail"
+beta = true
+common = false
+delivery_guarantee = "best_effort"
+egress_method = "streaming"
+function_category = "transmit"
+healthcheck = true
+input_types = ["log"]
+write_to_description = "[Papertrail](https://www.papertrail.com/)"
+
+<%= render("_partials/_component_options.toml", type: "sink", name: "papertrail") %>
+
+<%= render("_partials/_buffer_options.toml", namespace: "sinks.papertrail.options") %>
+
+[sinks.papertrail.options.endpoint]
+type = "string"
+common = true
+required = true
+examples = ["logs.papertrailapp.com:12345"]
+description = "The endpoint to stream logs to."
+
+[sinks.papertrail.options.encoding]
```

Comment:
This should default to `json`.

Reply:
I am not following the reason to make this a default?

Reply:
In #1915 we removed defaults where there was more than one option according to advice from @binarylogic

Reply:
It should be a case by case basis. Clearly Honeycomb should receive structured data. It doesn’t even have full text searching capabilities...

Reply:
This isn't honeycomb? To me here it totally makes sense to let the user choose since papertrail doesn't takes advantage of the message field in syslog.

According to @Hoverbear we are no longer settings defaults? https://github.com/timberio/vector/pull/1832#discussion_r391293379

Reply:
Ha, I'm losing my mind. Sure, I think for papertrail it makes sense to require it. For Honeycomb not so much.